File: ws/elbay/u3/ws24OOPConceptsZoo/OOPConceptsZoo_Java_DidierElbay/OOPConceptsZoo.java/NetBeansProjects/OOPConceptsZoo/src/main/java/ec/edu/espe/oopconceptszoo/model/Food.java
package ec.edu.espe.oopconceptszoo.model;

/**
 * Represents a type of food given to farm animals.
 *
 * @author Didier Elbay
 */
public class Food {

    private int id;
    private String description;

    public Food(int id, String description) {
        this.id = id;
        this.description = description;
    }

    public int getId()             { return id; }
    public String getDescription() { return description; }
}

File: ws/elbay/u3/ws24OOPConceptsZoo/OOPConceptsZoo_Python_DidierElbay/OOPConceptsZoo.py/model/food.py
# Author: Didier Elbay
# Food model - OOP Concepts Zoo Project


class Food:
    def __init__(self, id_food, description):
        self.id = id_food
        self.description = description

    def __str__(self):
        return f"Food({self.id}, {self.description})"
